wixcode-seo/wix-animations.service.json

```diff
@@ -0,0 +1,198 @@
+{ "name": "wix-animations",
+  "mixes": [],
+  "labels":
+    [ "changed" ],
+  "location":
+    { "lineno": 1,
+      "filename": "animations.js" },
+  "docs":
+    { "summary": "The wix-animations module contains functionality for working with\n animations.",
+      "description":
+        [ "Animation sequences are composed using a timeline. A timeline defines what animations",
+          " are played on which page elements and when those animations are played.",
```

Comment:
I think you should explain the typical process - build a timeline, add animations, play. Took me some time to figure that out.